### DISCUSSION TITLE
Logistics point not working?

### DISCUSSION THREAD

**Discussion by ghost** [Vdauphin/HeartsAndMinds#1127] [Question and answer]
Hello, after completing a side missions that tasks one to bring the dog tag, or an enemy officer, back to the logistics point, nothing happens and theres no prompts. Am I missing something here? My friends and I can't figure it out and would like to do some side missions. Assets listed/attached below: 

![20210629163251_1](https://user-images.githubusercontent.com/86382884/123880658-fb536380-d8f7-11eb-94c0-75da6ba5ad08.jpg)
![20210629163220_1](https://user-images.githubusercontent.com/86382884/123880662-fc849080-d8f7-11eb-9b49-4d58feb34a39.jpg)
![20210629163224_1](https://user-images.githubusercontent.com/86382884/123880663-fd1d2700-d8f7-11eb-9ca4-9603f7fb4133.jpg)
![20210629163230_1](https://user-images.githubusercontent.com/86382884/123880665-fdb5bd80-d8f7-11eb-95d7-0c4ecfcc99b6.jpg)
![20210629163235_1](https://user-images.githubusercontent.com/86382884/123880668-fe4e5400-d8f7-11eb-9adc-8070f77cc624.jpg)

**Comment by Vdauphin** [comment]
Hello,

Looks fine from here, may be you took the wrong dog tag/officer

Try your mission in Multiplayer menu of A3
- activate debug mode in mission parameter
- start the mission 
- execute server side (doesn't matter as you are in Multiplayer):
```sqf
[false, "btc_fnc_side_kill"] spawn btc_fnc_side_create;
```
- try to complate the mission (you are in god mode and can teleport by clicking on the map)

Cheers
